pkg/controller/operators/olm/operator.go

```diff
@@ -86,6 +92,7 @@ type Operator struct {
 	clientAttenuator      *scoped.ClientAttenuator
 	serviceAccountQuerier *scoped.UserDefinedServiceAccountQuerier
 	clientFactory         clients.Factory
+	isCopiedCSVsEnabled   bool
```

Comment:
nit: would areCopiedCSVsEnabled read better?

Reply:
What about hasCopiedCSVsEnabled? Then it would read:

`if a.hasCopiedCSVsEnabled {`

Reply:
`if a.copiedCSVsAreEnabled {...}`?

Reply:
I like `copiedCSVsAreEnabled`